eth/filters/filter_system.go

```diff
@@ -66,18 +68,21 @@ const (
 	logsChanSize = 10
 	// chainEvChanSize is the size of channel listening to ChainEvent.
 	chainEvChanSize = 10
+	// stateChangeChanSize is the size of the channel listening to StateChangeEvent.
+	stateChangeChanSize = 10
 )
 
 type subscription struct {
-	id        rpc.ID
-	typ       Type
-	created   time.Time
-	logsCrit  ethereum.FilterQuery
-	logs      chan []*types.Log
-	hashes    chan []common.Hash
-	headers   chan *types.Header
-	installed chan struct{} // closed when the filter is installed
-	err       chan error    // closed when the filter is uninstalled
+	id                  rpc.ID
+	typ                 Type
+	created             time.Time
+	logsCrit            ethereum.FilterQuery
```

Comment:
super minor but I wonder if we might want to rename this property if the FilterQuery can be used for more than logs